functional_tests/testdata/splunk/Makefile

```diff
@@ -0,0 +1,11 @@
+# Default image repository
+IMAGE_REPO_QUAY ?= quay.io/splunko11ytest/splunk
+IMAGE_REPO_DOCKER ?= splunk/splunk
+IMAGE_VERSION ?= 8.2.0
+
+# Note: The Splunk image does not support ARM architecture at this time.
+# Use Skopeo to copy the image between registries, allowing local developers with ARM machines to execute this action.
+.PHONY: push
+push:
+	# Copy the image from Docker Hub to Quay
+	skopeo copy --all docker://$(IMAGE_REPO_DOCKER):$(IMAGE_VERSION) docker://$(IMAGE_REPO_QUAY):$(IMAGE_VERSION)
```

Comment:
Are there any specific advantages to pulling from quay? Are we hitting the rate limit from dockerhub in our pipelines? I haven't noticed any ratelimiting so curious why are we copying to quay

Reply:
Directly messaged you